articles/dms/tutorial-mysql-azure-mysql-offline-portal.md

```diff
@@ -0,0 +1,254 @@
+---
+title: "Tutorial: Migrate MySQL offline to Azure Database for MySQL"
```

Comment:
Other tutorials follow this pattern for title: Migrate MySQL to Azure Database for MySQL offline using DMS

Reply:
done